src/test/java/org/jenkinsci/plugins/workflow/cps/CpsFlowDefinitionTest.java

```diff
@@ -0,0 +1,43 @@
+package org.jenkinsci.plugins.workflow.cps;
+
+import org.junit.Test;
+
+/**
+ * @author Kohsuke Kawaguchi
+ */
+public class CpsFlowDefinitionTest extends AbstractCpsFlowTest {
+    /**
+     * CpsFlowDefinition's simplest possible test.
+     */
+    @Test
+    public void simplestPossibleTest() throws Exception {
+        CpsFlowDefinition flow = new CpsFlowDefinition(
+                "def sqrt(int x) { return Math.sqrt(x); }\nfor (int i=0; i<10; i++)\nsqrt(i);",
+                false);
+
+        createExecution(flow);
+        exec.start();
+        exec.waitForSuspension();
+
+        assert exec.isComplete();
```

Comment:
Per [Error Prone's `UseCorrectAssertInTests` check](https://errorprone.info/bugpattern/UseCorrectAssertInTests):

> Java assert statements may or may not be evaluated depending on runtime flags to the JVM invocation. When used in tests, this means that the test assertions may not be checked, and a test may pass when it should actually fail. To avoid this, use one of the assertion libraries that are always enabled, such as JUnit’s `org.junit.Assert` or Google’s Truth library.

In this case, I'd suggest adding this import …

```java
import static org.junit.Assert.assertTrue;
```

… and then changing this line to:

```java
assertTrue(exec.isComplete());
```